dodo/app.py

```diff
@@ -275,6 +276,12 @@ def num_panels(self) -> int:
 
         return self.tabs.count()
 
+    def refresh_tab_titles(self) -> None:
+        for i in range(self.num_panels()):
+            w = self.tabs.widget(i)
+            if isinstance(w, panel.Panel):
+                self.tabs.setTabText(i, w.title())
```

Comment:
WDYT about

```suggestion
            assert isinstance(w, panel.Panel), w
            self.tabs.setTabText(i, w.title())
```

given that we never should have anything other than panels in there (I think)?

Reply:
We don't make this assumption anywhere so far (I stole this pattern from elsewhere in that file)